music/src/core/recorder.ts

```diff
@@ -82,6 +122,57 @@ export class Recorder {
     return this.recording;
   }
 
+  /**
+   * Changes the tempo of the click loop.
+   *
+   * @param qpm The new qpm to use.
+   */
+  setTempo(qpm: number) {
+    this.config.qpm = qpm;
+    if (Tone.Transport.state === 'started') {
+      Tone.Transport.bpm.value = qpm;
+    }
+  }
+
+  /**
+   * Whether to play a click track while recording. If the recorder is
+   * already recording, changes will not take place until it is restarted.
+   *
+   * @param playClick True if it should play a click track.
+   */
+  enablePlayClick(playClick: boolean) {
+    this.config.playClick = playClick;
+  }
+
+  /**
+   * Whether to play a count in at the beginning of the recording. If the
+   * recorder is already recording, changes will not take place until it
+   * is restarted.
+   *
+   * @param countIn True if it should play a count in.
+   */
+  enablePlayCountIn(countIn: boolean) {
+    this.config.playCountIn = countIn;
+  }
+
+  private initClickLoop() {
+    let clickStep = 0;
+    this.clickLoop = new Tone.Loop((_:number) => {
+      // TODO(notwaldorf): It would be nice if this took into account a
+      // time signature.
+      if (clickStep % 4 === 0) {
+        this.loClick.triggerAttack('G5');
```

Comment:
Using the passed in number parameter (_:number)  as the second argument to triggerAttack will give you much more accurate timing.

Reply:
I'm not sure I understand how to get "every 4 beats" from that, though